irods/user.py

```diff
@@ -9,6 +9,13 @@ class Bad_password_change_parameter(Exception): pass
 
 class iRODSUser(object):
 
+    def remove_quota(self, resource = 'total'):
+        self.manager.remove_quota(self.name, resource = resource)
+
+    # TODO: remove this in branch 2.x
```

Comment:
Do we have an issue for this?

Reply:
We do now: https://github.com/irods/python-irodsclient/issues/482